api/ws/workers/middlewares/handshake.js

```diff
@@ -72,11 +72,17 @@ var middleware = {
 		return function(headers, cb) {
 			z_schema.validate(headers, definitions.WSPeerHeaders, error => {
 				if (error) {
+					var errorDescription;
```

Comment:
Why not `let`?

Reply:
To keep consistency within the file module. This file uses `var` everywhere.